### PR TITLE
Initial draft impl of generate-v1-only-permissions action

### PR DESCRIPTION
**Pull request by wscalf**

### CODE REVIEW

File: generate-v1-only-permissions/action.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+runs:
+  using: docker
+  image: Dockerfile
+  args:
+    - "-ksl"
+    - ${{ inputs.ksl }}
+    - "-rbac-permissions-json"
+    - ${{ inputs.rbac_permissions }}
+inputs:
+  ksl_src:
```
Comment (lennysgarage) [inline generate-v1-only-permissions/action.yml:12]: I believe this should just be `ksl` based on `${{ inputs.ksl }}`

**Comment by lennysgarage** [comment]
LGTM!

**Review by astrozzc — APPROVED** [review]
LGTM

File: generate-v1-only-permissions/v2/V2.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+func WriteV1OnlyPermissionsFile(kslPath string, perms []string) error {
+	refs := []*intermediate.ExtensionReference{}
+
+	for _, perm := range perms {
+		refs = append(refs, &intermediate.ExtensionReference{Namespace: "rbac", Name: "add_v1only_permission", Params: map[string]string{"perm": perm}})
+	}
+
+	ns := &intermediate.Namespace{Name: "rbac_v1_permissions", Imports: []string{"rbac"}, ExtensionReferences: refs}
+
+	f, err := os.OpenFile(filepath.Join(kslPath, "src", V1OnlyPermissionsFile), os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0644)
```
[the remark this answers is not in the document]
Comment (wscalf) [reply generate-v1-only-permissions/v2/V2.go:43]: Good catch! Fixed

**Pull request merged**